setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
 	name="hera_gpu",
-	version="2018.08.3",
+	version="2019.06.1",
```

Comment:
version and author should be updated probably

Reply:
Done